testsuite/MDAnalysisTests/coordinates/base.py

```diff
@@ -1,12 +1,15 @@
-import MDAnalysis as mda
 import numpy as np
 from six.moves import zip
-from MDAnalysis.coordinates.base import Timestep
-
-from numpy.testing import (assert_equal, assert_raises, assert_almost_equal,
-                           assert_array_almost_equal, raises)
+from nose.plugins.attrib import attr
```

Comment:
Just an FYI. I tried to sort imports like this separated by a line.

1. General Python modules
2. Testing modules
3. MDAnalysis modules

Not really important but I personally try follow this. That makes it easier for me to spot relevant imports when errors occur